utils/githooks/README.md

```diff
@@ -0,0 +1,19 @@
+# Git hooks
```

Comment:
(lint) Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]